recharges/tasks.py

```diff
@@ -297,17 +300,21 @@ def run(self, recharge_id, **kwargs):
             if hs_recharge_status_cd == 3:
                 # Success
                 recharge.status = 2
+                recharge.status_message = "Recharge successful"
```

Comment:
I think we should just put the message from Hotsocket in here - and the other places